src/people/main/FocusView.tsx

```diff
@@ -60,6 +61,19 @@ function FocusedView(props: FocusViewProps) {
 
   const isTorSave = canEdit && main.isTorSave();
 
+  const getOrg = async () => {
+    if (uiStore.meInfo?.id) {
+      const res = await main.getUserDropdownOrganizations(uiStore.meInfo?.id);
+    }
+  };
+  getOrg();
+
+  function getUUIDFromURL(url: string) {
+    const parts = url.split('/');
```

Comment:
lets check if the url matches this pattern first and then return the last part
`{url}/org/bounties/{uuid}`